src/oracles.jl

```diff
@@ -303,3 +303,61 @@ function compute_extreme_point(
     end
     return storage
 end
+
+"""
+"""
+struct ChasingGradientLMO{LMO<:LinearMinimizationOracle,T} <: LinearMinimizationOracle
+    inner::LMO
+    max_rounds_number::Int
+    improv_tol::T
+end
+
+"""
+"""
+function compute_extreme_point(lmo::ChasingGradientLMO, direction; x, kwargs...)
+    d = 0 * direction
+    Λ = 0
+    #@debug x
+    #@debug direction
+    if norm(direction) == 0
+        return compute_extreme_point(lmo.inner, direction, x=x, kwargs...)
+    end
+    flag = true
+    function align(d1, d2)
+        if norm(d1) == 0 || norm(d2) == 0
```

Comment:
the align function will be type-unstable here (a different type will be returned in the two branches)